jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="./test/e2e_node/jenkins/e2e-node-jenkins.sh"
```

Comment:
`W1219 13:56:36.261] /workspace/node-runner.sh: line 31: test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/jenkins-ci.properties: No such file or directory`

I think I'm not in the correct directory..